server/apps/search/__init__.py

```diff
@@ -22,7 +21,7 @@ class SearchService(superdesk.Service):
     It can search ingest/content/archive/spike at the same time.
     """
 
-    available_repos = ('ingest', 'archive')
+    available_repos = ('ingest', 'archive', 'aapmultimedia')
```

Comment:
This should be resolved after the application loads the modules, basically a module should register a search service in the form of a datalayer/or simpler API if we are going to do only search actions against it and SD datalayer should just dynamically use them.